ReserveSpot.Services.Api/Controllers/HomeController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+
+namespace ReserveSpot.Services.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class HomeController : RESTFulController
+    {
+        [HttpGet]
+        public ActionResult<string> get() =>
+            Ok("Hurry Up! Reserve Your Spot. We are happy to see you playing sports.");
```

Comment:
Attach a screen shot that shows this message.

Reply:
![Home Controller Output](https://user-images.githubusercontent.com/95454126/155249091-2e9ab30c-70ed-413b-a471-c8f2f819fa54.png)